lib/cjs/generated/rum.d.ts

```diff
@@ -858,9 +858,9 @@ export interface CommonProperties {
          */
         session?: {
             /**
-             * Session plan: 1 is the 'lite' plan, 2 is the 'replay' plan
+             * Session plan: 1 is the 'lite' plan, 2 is the 'premium' plan, 3 is the 'pro' plan, 4 is the 'replay' plan
              */
-            plan: 1 | 2;
+            plan: 1 | 2 | 3 | 4;
```

Comment:
**question:** replay was 2 and now it's 4, are we sure about this? It won't break existing behaviour?
**Disclaimer:** I have no idea what the new one are

Reply:
The only impact here is on the comment.
'replay' for 2 was outdated and we never fixed it to premium.
To be noted: 'pro' and 'replay' namings may still change, not a big deal for the format structure though.

Reply:
it should have been but a bit late now 🙂